src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppAppRunner/Generated/Recipe.cs

```diff
@@ -0,0 +1,154 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using Amazon.CDK;
+using Amazon.CDK.AWS.AppRunner;
+using Amazon.CDK.AWS.IAM;
+using Amazon.CDK.AWS.ECR;
+using Amazon.CDK.AWS.ECS;
+using AWS.Deploy.Recipes.CDK.Common;
+using AspNetAppAppRunner.Configurations;
+
+using CfnService = Amazon.CDK.AWS.AppRunner.CfnService;
+using CfnServiceProps = Amazon.CDK.AWS.AppRunner.CfnServiceProps;
+
+// This is a generated file from the original deployment recipe. It is recommended to not modify this file in order
+// to allow easy updates to the file when the original recipe that this project was created from has updates.
+// To customize the CDK constructs created in this file you should use the AppStack.CustomizeCDKProps() method.
+
+namespace AspNetAppAppRunner
+{
+    using static AWS.Deploy.Recipes.CDK.Common.CDKRecipeCustomizer<Recipe>;
+
+    public class Recipe : Construct
+    {
+        public CfnService? AppRunnerService { get; set; }
+
+        public IRole? ServiceAccessRole { get; set; }
+
+        public IRole? TaskRole { get; set; }
+
+        public Recipe(Construct scope, IRecipeProps<Configuration> props)
+       : base(scope, "Recipe")
```

Comment:
There should be a comment explaining why the construct id is constant here. And changing it will impact the stack.

Reply:
Done